server/chat-plugins/poll.js

```diff
@@ -218,6 +222,7 @@ exports.Poll = Poll;
 /** @type {ChatCommands} */
 const commands = {
 	poll: {
+		anoncreate: 'new',
```

Comment:
That means you can't create anonymous HTML polls.

You'll probably need to make another alias (`anonhtmlcreate`) and handle it accordingly.

Reply:
Yeah, I saw that, just didnt think it mattered.